README.md

```diff
@@ -0,0 +1,38 @@
+# Mobtime Web UI
+A simple front end for MobTime; a collaborative mobbing timer for desktop and mobile.
+
+## Overview
+
+Lightweight front end for MobTime. Built using [Hyperapp](https://github.com/jorgebucaran/hyperapp) & [Vite](https://vitejs.dev)
+
+There are two main pages: the homepage and the timer page
+
+- Homepage: `src/index.html`
+
+  <img width="300" alt="Timer page" src="https://imgur.com/L2LGR7G.jpg" />
+
+- Timer Page: `src/timer/index.html`
+
+  <img width="300" alt="Timer page" src="https://imgur.com/FVweaWL.jpg" />
+
+## Running Locally
+Vite is used to run a development server locally. 
+
+1. clone this repo &rarr; `git clone https://github.com/mobtimeapp/mobtime-web.git`
+2. Install dependencies &rarr; `npm i`
+3. Build CSS &rarr; `npm run tailwind`
+4. Run Vite dev server &rarr; `npm run dev`
```

Comment:
Can you update this to `npm start` please.

Reply:
Yup - nice catch, thanks.